files/zh-cn/learn/javascript/first_steps/silly_story_generator/index.md

```diff
@@ -187,6 +187,7 @@ slug: Learn/JavaScript/First_steps/Silly_story_generator
   document.querySelector("html").style.backgroundColor = "red";
   ```
 
+- 公制单位与美制单位之间的转换公式 `华氏度 = 32+ 摄氏度× 1.8` `一磅 = 0.45359千克`
```

Comment:
Did not exist in en-US content. Please remove it, as we only do [translations](https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md)